### PR TITLE
Improvements 02

### PR DESCRIPTION
**Pull request by MatthewCroughan**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 694eac3c10814646a1ec46b4d4536bebc68eeb8f are browesable at: https://ci.foundries.io/projects/fio-docs/builds/720/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for efb50389556208eb71b9fe712a941a3ab00e3caf are browesable at: https://ci.foundries.io/projects/fio-docs/builds/722/docs/artifacts/html/index.html

File: source/user-guide/configure-lmp/index.rst [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+LMP_HOSTNAME_NETDEVICE
+  *if* using ``mac`` mode, choses what network interface on devices to retrieve
+  a mac address from.
+
+  **Default:** ``eth0``
+
+  **Example Value:** ``eth0`` or ``wlan0``
+
+.. _meta-lmp: https://github.com/ricardosalveti/meta-lmp/tree/master
+.. _meta-lmp-base: https://github.com/ricardosalveti/meta-lmp/tree/master/meta-lmp-base
```
Comment (ricardosalveti) [inline source/user-guide/configure-lmp/index.rst:246]: Upstream for meta-lmp is https://github.com/foundriesio/meta-lmp

**Comment by doanac** [comment]
Docs for fd4da86fd15f724090d550bb22eccce6a28229b3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/723/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c4217d7b6e2b32f8f41228fdfe9c184b50903535 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/724/docs/artifacts/html/index.html

File: source/user-guide/configure-lmp/index.rst (hunk 1 above)
Comment (MatthewCroughan) [reply source/user-guide/configure-lmp/index.rst]: Didn't spot that, was stuck browsing your repo when I was making this :) c4217d7

File: source/user-guide/configure-lmp/index.rst [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+.. sidebar:: How to Enable
+
+   #. Populate the recipe variables in:
+   
+      **meta-subscriber-overrides/conf/machine/include/lmp-factory-custom.inc**
+   
+      .. code-block::
+   
+         IMAGE_INSTALL_append = " lmp-auto-hostname"
```
Comment (ricardosalveti) [inline source/user-guide/configure-lmp/index.rst:40]: No need for IMAGE_INSTALL_append as later we're adding the recipe at lmp-factory-image.

Comment (MatthewCroughan) [reply source/user-guide/configure-lmp/index.rst]: db03780

**Comment by doanac** [comment]
Docs for db03780ce7d1803bb462a267f13f62affb389b5b are browesable at: https://ci.foundries.io/projects/fio-docs/builds/725/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for dc3d440ffce726906ef6585868c3ce3a026693bf are browesable at: https://ci.foundries.io/projects/fio-docs/builds/726/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for e40424d9a4e323397e4f9fb340ca5453a84c516d are browesable at: https://ci.foundries.io/projects/fio-docs/builds/727/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 71c5b5d2ea43cb8d4d2f5e41b4fe31a77237c1d7 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/728/docs/artifacts/html/index.html

**Pull request merged**